spec/01-lexical-syntax.md

```diff
@@ -6,13 +6,11 @@ chapter: 1
 
 # Lexical Syntax
 
-Scala programs are written using the Unicode Basic Multilingual Plane
-(_BMP_) character set; Unicode supplementary characters are not
-presently supported.  This chapter defines the two modes of Scala's
-lexical syntax, the Scala mode, and the _XML mode_. If not
-otherwise mentioned, the following descriptions of Scala tokens refer
-to _Scala mode_, and literal characters ‘c’ refer to the ASCII fragment
-`\u0000` – `\u007F`.
+Scala source text consists of Unicode characters.
```

Comment:
Consider writing "Unicode code points" instead of "Unicode characters". Unicode has, on purpose, no notion of "character" (the closest that comes to the human notion of character is in fact a Unicode Extended Cluster).

Reply:
Thanks, it crossed my mind. I'll reconsider. "Scala code is Unicode code points!" It's points of code!